data_structures/arrays/pascals_triangle.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+
+class Solution:
+    def generate(self, numRows: int) -> List[List[int]]:
```

Comment:
No need to create this class.
```suggestion
def generate(self, numRows: int) -> List[List[int]]:
```